src/main/java/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction.java

```diff
@@ -23,32 +34,40 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.concurrent.ExecutionException;
+import jenkins.model.Jenkins;
 import net.sf.json.JSONObject;
+import org.jenkins.ui.icon.IconSpec;
+import org.jenkinsci.plugins.pipeline.modeldefinition.actions.RestartDeclarativePipelineAction;
+import org.jenkinsci.plugins.workflow.cps.replay.ReplayAction;
 import org.jenkinsci.plugins.workflow.flow.FlowExecution;
 import org.jenkinsci.plugins.workflow.graph.FlowNode;
 import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 import org.kohsuke.stapler.HttpResponse;
 import org.kohsuke.stapler.StaplerRequest2;
 import org.kohsuke.stapler.StaplerResponse2;
 import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.bind.JavaScriptMethod;
+import org.kohsuke.stapler.interceptor.RequirePOST;
 import org.kohsuke.stapler.verb.GET;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class PipelineConsoleViewAction extends AbstractPipelineViewAction {
+public class PipelineConsoleViewAction implements Action, IconSpec {
     public static final long LOG_THRESHOLD = 150 * 1024; // 150KB
     public static final String URL_NAME = "pipeline-overview";
 
     private static final Logger logger = LoggerFactory.getLogger(PipelineConsoleViewAction.class);
-    private final WorkflowRun target;
-    private final PipelineStepApi stepApi;
-
     private static final ObjectMapper MAPPER = new ObjectMapper();
 
+    private final transient PipelineGraphApi graphApi;
+    private final transient WorkflowRun run;
```

Comment:
do these need to be `transient`? they were in the base but the `PipelineStepApi` isn't and the other copy of the field, `target`, wasn't originally